.github/workflows/_get_next_version.yml

```
@@ -0,0 +1,47 @@
+    outputs:
+      next-semver: ${{steps.semver.outputs.semver}}
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js 20.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - name: Get next semver
+        id: semver
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:6: Double quote to prevent globbing and word splitting

(shellcheck)